### DISCUSSION TITLE
supabase-js insert default values

### DISCUSSION THREAD

**Discussion by ethicnology** [supabase/supabase#1792] [Questions] [answered]
Hi,
What is the **supabase-js** equivalent to 
```sql
INSERT INTO table DEFAULT VALUES;
```

I tried with no succeeding
```ts
const { data, error } = await SUPABASE.from('gear').insert([])
```

Is it even possible ?

**Comment by soedirgo** [comment, marked as answer]
Hey @ethicnology, can you try `await supabase.from('gear').insert([{}])`? `insert`ing with an array is a bulk insert, and an empty array means there's nothing to insert.

**Reply by ethicnology** [reply]
It doesn't work
```
{
  details: 'unexpected end of input expecting field name (* or [a..z0..9_])',
  message: '"failed to parse columns parameter ()" (line 1, column 1)'
}
```

**Reply by soedirgo** [reply]
Can you see if updating supabase-js fixes it? There was a bug in an older version that gives that error.

**Reply by ethicnology** [reply]
Thanks it's works since yesterday version **supabase 1.13.1**
